submissions/oluwatosinOke/phpAndDatabases/customerlist.php

```diff
@@ -0,0 +1,35 @@
+<?php
+require_once("cookie.php");
+try {
+  	$sql = "SELECT * FROM customers";
```

Comment:
What is on your `cookie.php` is the same thing you are doing here.

That aside, the idea of this task is to store the result returned from this SQL query in a cookie, Each time I reload this page, I don't want to always run this SQL query but instead, I want to get it from the data stored in the cookie as long as the cookie has not expired. If the cookie is has expired then run the SQL query

With your current implementation, each time I reload this page, I am always executing this SQL query.